CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ description: |-
 
 # VKCS Provider's changelog
 
+#### v0.2.0 (unreleased)
+- Add vkcs_db_datastore_capabilities datasource
```

Comment:
Add datasource for DB capabilities

Reply:
i expected you either change other line similar way or mention here capabilities and config parameters